keyhub/resource_group.go

```diff
@@ -0,0 +1,355 @@
+package keyhub
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	keyhubclient "github.com/topicuskeyhub/go-keyhub"
+	"strconv"
+	"strings"
+
+	keyhubmodel "github.com/topicuskeyhub/go-keyhub/model"
+)
+
+func resourceGroup() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceGroupCreate,
+		ReadContext:   dataSourceGroupRead,
+		UpdateContext: resourceGroupUpdate,
+		DeleteContext: resourceGroupDelete,
+		Schema:        GroupResourceSchema(),
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceGroupImportContext,
+		},
+	}
+}
+
+func resourceGroupImportContext(ctx context.Context, d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
+
+	grpUuid, err := uuid.Parse(d.Id())
+	if err != nil {
+		return nil, fmt.Errorf("`%s` is not a valid uuid", d.Id())
+	}
+
+	d.Set("uuid", grpUuid.String())
```

Comment:
Dit kan niet. d.Id() is geen UUID maar het database ID.
Zie:

[Resource Create](https://github.com/topicuskeyhub/terraform-provider-keyhub/blob/a34c839e3fde504010528047bd0c20b1e5abd2f0/keyhub/resource_group.go#L307)
[Data Read](https://github.com/topicuskeyhub/terraform-provider-keyhub/blob/a34c839e3fde504010528047bd0c20b1e5abd2f0/keyhub/data_source_group.go#L217)